src/main/java/org/heigit/ohsome/ohsomeapi/output/dataaggregationresponse/DefaultAggregationResponse.java

```diff
@@ -4,15 +4,18 @@
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import io.swagger.annotations.ApiModelProperty;
 import org.geojson.Feature;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 /**
  * Represents the outer JSON response object for the data aggregation requests that do not use the
  * /groupBy resource. It contains attribution info, the version of the api, optional
- * {@link org.heigit.ohsome.ohsomeapi.output.dataaggregationresponse.Metadata
- * Metadata} and the
+ * {@link org.heigit.ohsome.ohsomeapi.output.dataaggregationresponse.Metadata Metadata} and the
  * {@link org.heigit.ohsome.ohsomeapi.output.dataaggregationresponse.elements.ElementsResult
  * ElementsResult} objects.
  */
+@Getter
+@NoArgsConstructor
```

Comment:
I think here you'd also have to define _access = AccessLevel.PRIVATE_ right?

Reply:
yes, you're right